_includes/v20.1/misc/movr-workflow.md

```diff
@@ -1,49 +1,76 @@
-The workflow for MovR is as follows (with approximations of the corresponding SQL for each step):
+The workflow for MovR is as follows:
 
-1. A user loads the app and sees the 25 closest vehicles:
+1. A user loads the app and sees the 25 closest vehicles.
+    
+    For example:
 
+    {% include copy-clipboard.html %} 
     ~~~ sql
-    > SELECT id, city, status, ... FROM vehicles WHERE city = <user location>
-    ~~~
-
-2. The user signs up for the service:
+    > SELECT id, city, status FROM vehicles WHERE city='amsterdam' limit 25;
+    ~~~   
 
-    ~~~ sql
-    > INSERT INTO users (id, name, address, ...) VALUES ...
-    ~~~
+2. The user signs up for the service.
 
-3. In some cases, the user adds their own vehicle to share:
+    For example:
 
+    {% include copy-clipboard.html %}
     ~~~ sql
-    > INSERT INTO vehicles (id, city, type, ...) VALUES ...
-    ~~~
+    > INSERT INTO users (id, name, address, city, credit_card) 
+      VALUES ('66666666-6666-4400-8000-00000000000f', 'Mariah Lam', '88194 Angela Gardens Suite 60', 'amsterdam', '123245696');
+    ~~~      
+    
+    {{site.data.alerts.callout_info}}Usually for [Universally Unique Identifier (UUID)](uuid.html#create-a-table-with-auto-generated-unique-row-ids) you would need to generate it automatically but for the sake of this follow up we will use predetermined UUID to keep track of them in our examples.{{site.data.alerts.end}}
```

Comment:
> [Universally Unique Identifier (UUID)](uuid.html#create-a-table-with-auto-generated-unique-row-ids)

This isn't generating a link properly in the HTML (this happens with markdown-styled links in notes).

Could you change this to:

```
<a href="uuid.html#create-a-table-with-auto-generated-unique-row-ids">Universally Unique Identifier (UUID)</a>
```

Reply:
I looked at other docs and it seemed to work fine the way you showed. Maybe it has something to do with the anchor. In any case, I changed it to the hyperlink format.